src/aliases.js

```diff
@@ -69,5 +69,6 @@ module.exports = {
   pk: "pumpkin king",
   pumpking: "pumpkin king",
   mod: "modded",
-  astral: "astral wolf"
+  astral: "astral wolf",
```

Comment:
make sure to add the `,` after each line (when you add a new one)

Reply:
Ok